docs/modules/ROOT/pages/index.adoc

```diff
@@ -323,13 +325,36 @@ The `signer_payload` data structure MUST be presented to be signed by the _<<_cr
 
 IMPORTANT: If a data hash assertion is being used, the _<<C2PA claim generator>>_ MUST also follow the process described in xref:_interaction_with_data_hash_assertion[xrefstyle=full].
 
+==== Calculating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim` value in `signer_payload`, it MUST be computed as follows:
+
+. Create a valid _<<C2PA claim>>_ data structure that includes completed references to all _<<_c2pa_assertion,C2PA assertions>>,_ *including* all *<<_identity_assertion,identity assertion>>.*
+. For the specific *<<_identity_assertion,identity assertion>>* being computed, replace the `hash` value with 32 zero-value (`0x00`) bytes.
```

Comment:
Is this always 32 byte long, or does it depend on the length of a configurable hash function?

Reply:
I saw later that SHA-256 is hardcoded. See my related comment suggesting we should perhaps make that a param for crypto agility.